jobs/config.json

```diff
@@ -6598,18 +6598,17 @@
     "args": [
       "--cluster=gke-large-deploy",
       "--deployment=gke",
-      "--down=false",
       "--extract=ci/latest",
       "--gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging",
       "--gcp-node-image=gci",
       "--gcp-project=k8s-scale-testing",
       "--gcp-zone=us-east1-a",
-      "--gke-create-command=container clusters create --quiet --max-nodes-per-pool=1000",
```

Comment:
Let's not remove this

Reply:
It's the default.

Reply:
Aah ok, didn't know that.

Reply:
looks like in kubetest default does not have `--max-nodes-per-pool=1000`? Or that's the default in gke?

Reply:
Yes... The gke documentation says the default value is 1000 - https://cloud.google.com/sdk/gcloud/reference/container/clusters/create